modules/gapi/include/opencv2/gapi/infer/bindings_onnx.hpp

```diff
@@ -0,0 +1,44 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level
+// directory of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2022 Intel Corporation
```

Comment:
@xiong-jie-y Are you sure about the last copyright line? If you don't work at Intel (email is not) not sure about then please remove this line (first 3 lines are enough).

Reply:
I see. Removed!